_source/_posts/2021-08-13-reactive-java.md

```diff
@@ -8,7 +8,8 @@ author: jimena-garbarino
 by: contractor
 communities: [java]
 description: "Reactor Schedulers for Spring Reactive Java Applications"
-tags: [spring-boot, spring-framework, spring-security, oidc, oauth2, reactive, webflux, blockhound, java]
+tags: [spring-boot, spring-framework, 
+x, blockhound, java]
```

Comment:
Did you mean to change this blog post? Looks like a mistake.

Reply:
Oops. Sorry. Yeah, totally a mistake.